### PR TITLE
[snap]: fix hardcoded app icon

### LINKED ISSUE #1514: Icon for Adwaita theme
Dears,
I'm using Ubuntu 23.10 with Adwaita (Default) theme.
using this theme I see a generic icon for App Center.
Who needs to create a dedicated icon for this application?
Thanks!

immagine

### PR DESCRIPTION
**Pull request by soumyaDghosh**
Fixes #1514 

image

### CODE REVIEW

**Comment by soumyaDghosh** [comment]
@BLKKKBVSIK @spydon can you look into this. This is relatively small PR.

**Comment by BLKKKBVSIK** [comment]
LGTM

**Review by BLKKKBVSIK — APPROVED** [review]

File: packages/app_center/assets/app-center.desktop [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Type=Application
 Version=1.0
 Exec=snap-store %U
-Icon=software-store
+Icon=bin/data/flutter_assets/assets/app-center.png
```
Comment (spydon) [inline packages/app_center/assets/app-center.desktop:5]: It seems like most snaps have these entries as absolute to their snap directory, could that be an option here?
The firmware-updater for example:
```
Icon=/snap/firmware-updater/109/meta/gui/icon.png
```

Comment (soumyaDghosh) [reply packages/app_center/assets/app-center.desktop:5]: It'll be rewritten by snapd, we don't need to worry about it.

image

Comment (spydon) [reply packages/app_center/assets/app-center.desktop:5]: Aah, super

**Review by spydon — APPROVED** [review]

**Comment by Feichtmeier** [comment]
I don't know if this was intended, but now the icon changed to a monochrome, half transparent icon from the previous yaru full color app icon

now
image

before
image

**Comment by soumyaDghosh** [comment]
@Feichtmeier that's probably not the monochrome icon, but the new icon. It's taken from here,

https://github.com/ubuntu/app-center/blob/main/packages/app_center/assets/app-center.png

Now, I am not sure, if using the `svg` version would fix it. And the screenshot I shared was not objected by anyone, so, probably intended

**Comment by AsciiWolf** [comment]
@soumyaDghosh I am not sure whether using a 90x90 px icon is a good idea. The svg one would probably be better. Or [this one](https://github.com/ubuntu/app-center/blob/main/packages/app_center/assets/2.0x/app-center.png) that has better resolution (although it still is a non-standard size) .

**Pull request merged**